apps/frontend/lib/frontend/web/utils.ex

```diff
@@ -59,4 +62,12 @@ defmodule Frontend.Utils do
   defp transform_to_date(date) do
     date
   end
+
+  @spec has_permission?(term, term) :: Boolean.t
+  def has_permission?(user, permission) do
+    {:ok, role} = Roles.get(id: user.role_id)
+    {:ok, permission} = Permissions.get(name: permission)
+    {:ok, rp} = RP.get_permission_with_role(role.id, permission.id)
+    if(rp, do: true, else: false )
```

Comment:
Modify `get_with_role` to include permissions so you can just enumerate list of permission and return true/false if permission is in list...